BPRBlazor/Program.cs

```diff
@@ -1,13 +1,23 @@
+using BPRBE.Config;
+using BPRBE.Models.Persistence;
+using BPRBE.Validators;
+using FluentValidation;
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 builder.Services.AddRazorPages();
 builder.Services.AddServerSideBlazor();
 builder.Services.AddHttpClient();
-
+// Services
 builder.Services.AddScoped<IHttpService, HttpService>();
-
+builder.Services.AddScoped<IMongoDependencyRepository, MongoDependencyRepository>();
+// Validators
+builder.Services.AddScoped<IValidator<ArchitecturalModel>, ArchitecturalModelValidator>();
+builder.Services.AddScoped<IValidator<ArchitecturalComponent>, ArchitecturalComponentValidator>();
+builder.Services.AddScoped<IValidatorService, ValidatorService>();
+// Database config
+builder.Services.Configure<DatabaseConfig>(builder.Configuration.GetSection(DatabaseConfig.Section));
```

Comment:
Could consider making extension methods on the IServiceCollection interface and have these register the services so you could just call builder.Services.AddValidators() and builder.Services.ConfigureSettings() rather than having these comments.